justega247/favorite-things
# #167131655: Setup the django project

#### What does this PR do?
- This PR is for the initial setup of the project upon which every other implementation will be made
#### Description of Task to be completed?
- Setup the projects database
- Install django
- Install djangorestframework
#### How should this be manually tested?
- In your terminal run go to the root directory of the project and run **python manage.py runserver**
- Then navigate to the url, you should see the django welcome page
#### What are the relevant pivotal tracker stories?
[#167131655](https://www.pivotaltracker.com/story/show/167131655)
